src/index.js

```diff
@@ -5,7 +5,7 @@ const todoContainer = document.querySelector(".todo-list");
 
 const generateMarkup = (data) => {
   return `<li class="todo" data-index="${data.index}" data-completed="${data.completed}"> 
-  <label for="${data.index}"> <i class="fa-regular fa-square"></i> </label>
+  <label for="${data.index}"> <input type="checkbox"> </label>
```

Comment:
- [ ] [REQUIRED] Great job with the implementation of the recommended changes, however, there is a hard requirement that has not been met. One of the project requirements states that: The task object's values for `completed` is updated upon user actions. This means whenever a user checks a checkbox, its status should update its value to true.
### Take a look at what you currently have in your local storage:
<img width="708" alt="Screenshot 2022-12-10 at 22 06 24" src="https://user-images.githubusercontent.com/58692590/206875619-7b46faf0-a548-44e2-9682-b56e0ac7b0b3.png">